### PR TITLE
Convert long running stress tests to projects

### PR DESCRIPTION
**Pull request by pcmoritz**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.

### CODE REVIEW

File: ci/long_running_tests/.rayproject/project.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: long-running-tests
```
Comment (ericl) [inline ci/long_running_tests/.rayproject/project.yaml]: Seems weird to put the project in a hidden .rayproj folder, why not remove the dot? Users will be confused by it being hidden.

Comment (pcmoritz) [reply ci/long_running_tests/.rayproject/project.yaml]: good point, I will change it in a follow up PR

File: ci/long_running_tests/.rayproject/cluster.yaml [hunk 2]
```diff
@@ -37,11 +37,6 @@ setup_commands:
     # Install latest TensorFlow
     - source activate tensorflow_p36 && conda remove -y --force wrapt || true
     - source activate tensorflow_p36 && pip install -U tensorflow==1.14
-    # Install nightly Ray wheels.
```
Comment (simon-mo) [inline ci/long_running_tests/.rayproject/cluster.yaml]: why are these removed?

Comment (simon-mo) [reply ci/long_running_tests/.rayproject/cluster.yaml]: ah i see you are moving it to project.yaml

File: ci/long_running_tests/README.rst [hunk 3]
```diff
@@ -1,29 +1,36 @@
...
 
-This directory contains scripts for starting long-running workloads which are
-intended to run forever until they fail.
+This directory contains the long-running workloads which are intended to run
+forever until they fail. To set up the project you need to run
+
+.. code-block:: bash
+
+    pip install any
+    any project create
```
Comment (simon-mo) [inline ci/long_running_tests/README.rst]: `ray project create`?

File: ci/long_running_tests/.rayproject/project.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      tmux: true
+
+  - name: check
+    command: if [ "{{load}}" == "True" ]; then uptime; else tmux capture-pane -p; fi
+    params:
+      - name: load
+        action: store_true
+        required: false
+        help: "If set, check the load of session(s)."
+      - name: logs
```
Comment (simon-mo) [inline ci/long_running_tests/.rayproject/project.yaml]: why are these parameterized? I would suggest just make them separate commands because they do different things. 
```
check-loads
tail-logs
```

File: ci/long_running_tests/README.rst (hunk 3 above)
Comment (pcmoritz) [reply ci/long_running_tests/README.rst]: This actually is needed to register it with the project database in the `any` command.

File: ci/long_running_tests/.rayproject/project.yaml (hunk 4 above)
Comment (pcmoritz) [reply ci/long_running_tests/.rayproject/project.yaml]: fixed

**Review by simon-mo — APPROVED** [review]
lgtm, had few questions about `any`, will discuss offline
